Tests/RedisKitTests/RedisKitTests.swift

```diff
@@ -31,6 +151,263 @@ final class RedisKitTests: XCTestCase {
+
+        let addResp1 = try self.client.sadd([RESPValue(bulk: dataSet[0])], to: "set1").wait()
+        XCTAssertEqual(addResp1, 1)
+        let addResp2 = try self.client.sadd(
+            [RESPValue(bulk: dataSet[1]), RESPValue(bulk: dataSet[2]), RESPValue(bulk: dataSet[3])],
+            to: "set1"
+        ).wait()
+        XCTAssertEqual(addResp2, 3)
+        let addResp3 = try self.client.sadd([RESPValue(bulk: dataSet[1])], to: "set1").wait()
+        XCTAssertEqual(addResp3, 0)
+
+        let countResp = try self.client.scard(of: "set1").wait()
+        XCTAssertEqual(countResp, 4)
+
+        let membersResp = try self.client.smembers(of: "set1").wait().map { $0.string! }
+        XCTAssertTrue(membersResp.allSatisfy { dataSet.contains($0) })
+
+        let isMemberResp1 = try self.client.sismember(RESPValue(bulk: dataSet[0]), of: "set1").wait()
+        XCTAssertTrue(isMemberResp1)
+        let isMemberResp2 = try self.client.sismember(RESPValue(bulk: "Vapor"), of: "set1").wait()
+        XCTAssertFalse(isMemberResp2)
+
+        let randResp1 = try self.client.srandmember(from: "set1").wait()
+        XCTAssertTrue(dataSet.contains(randResp1[0].string!))
+        let randResp2 = try self.client.srandmember(from: "set1", max: 2).wait()
+        XCTAssertTrue(randResp2.allSatisfy { dataSet.contains($0.string!) })
+        let randResp3 = try self.client.srandmember(from: "set1", max: 5).wait()
+        XCTAssertTrue(randResp3.count == 4)
+        let _ = try self.client.sadd([RESPValue(bulk: "Vapor"), RESPValue(bulk: "Redis")], to: "set2").wait()
+        let randResp4 = try self.client.srandmember(from: "set2", max: -3).wait()
+        XCTAssertTrue(randResp4.count == 3)
+        let randResp5 = try self.client.srandmember(from: "set2", max: 3).wait()
+        XCTAssertTrue(randResp5.count == 2)
+
+        #warning("new client.spop() returns [RESPValue] instead of single value")
+        /*let popResp = try redis.spop("set1").wait().string!
+        XCTAssertTrue(dataSet.contains(popResp))
+        XCTAssertEqual(try redis.scard("set1").wait(), 3)
+
+        let itemToRemove = dataSet.first(where: { $0 != popResp })!
+        let remResp1 = try redis.srem("set1", items: [RedisData(bulk: itemToRemove)]).wait()
+        XCTAssertEqual(remResp1, 1)
+        let remResp2 = try redis.srem("set1", items: [RedisData(bulk: "Vapor")]).wait()
+        XCTAssertEqual(remResp2, 0)
+        let remainingToRemove = dataSet.filter({ $0 != popResp && $0 != itemToRemove }).map { RedisData(bulk: $0) }
+        let remResp3 = try redis.srem("set1", items: remainingToRemove).wait()
+        XCTAssertEqual(remResp3, 2)*/
+    }
+
+    func testSortedSetCommands() throws {
+        let _ = try self.client.send(command: "FLUSHALL").wait()
+
+        let dataSet = [("1", RESPValue(bulk: "data1")),("2", RESPValue(bulk: "data2")),("4", RESPValue(bulk: "data3"))]
+
+        #warning("zadd declaration has changed")
```

Comment:
we should add an issue for this one, too 👍

Reply:
What issue? For a variadic overload, or?

Reply:
I was simply not sure what to do here to be honest, seems the api changed from strings to a double? I'm not familiar enough with Redis to see what that change entailed and how to adapt to it :)

Reply:
Yeah, this is just going to be a breaking API change that users will need to adapt to.

We'll probably want to capture all these changes in a migration doc...

Reply:
Submitted an issue in the vapor/redis repo to track this: https://github.com/vapor/redis/issues/152